test/RequestSender.spec.ts

```diff
@@ -1,44 +1,72 @@
 // @ts-nocheck
 import {expect} from 'chai';
+import nock = require('nock');
+
 import {
   StripeAuthenticationError,
   StripeConnectionError,
   StripeError,
   StripeIdempotencyError,
   StripePermissionError,
   StripeRateLimitError,
+  StripeUnknownError,
+  TemporarySessionExpiredError,
 } from '../src/Error.js';
-import {HttpClientResponse} from '../src/net/HttpClient.js';
 import {RequestSender} from '../src/RequestSender.js';
+import {ApiVersion} from '../src/apiVersion.js';
+import {HttpClientResponse} from '../src/net/HttpClient.js';
 import {
   FAKE_API_KEY,
   getSpyableStripe,
   getTestServerStripe,
 } from './testUtils.js';
-import nock = require('nock');
 
 const stripe = getSpyableStripe();
 
 describe('RequestSender', () => {
   const sender = new RequestSender(stripe, 0);
 
   describe('_makeHeaders', () => {
-    it('sets the Authorization header with Bearer auth using the global API key', () => {
-      const headers = sender._makeHeaders(null, 0, null);
-      expect(headers.Authorization).to.equal(`Bearer ${FAKE_API_KEY}`);
-    });
-    it('sets the Authorization header with Bearer auth using the specified API key', () => {
-      const headers = sender._makeHeaders('anotherFakeAuthToken', 0, null);
-      expect(headers.Authorization).to.equal('Bearer anotherFakeAuthToken');
-    });
```

Comment:
TODO: Check why these tests were removed

Reply:
Because the code for that moved to the authenticator and is tested later in the file here